code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -510,6 +520,7 @@
 	color = "#CF3600"
 	metabolization_rate = 0.1
 	penetrates_skin = 1
+	taste_message = "almonds"
```

Comment:
Fun fact, you ingest cyanide on a near daily basis. It's only really dangerous in extreme dosages. Note that we're talking extreme in the sense of poisons here though. It'll still only take about a half a teaspoon to cause any real harm, though even then, you'd still probably recover if you got medical treatment.